selfie.c

```diff
@@ -889,7 +889,7 @@ uint64_t MAX_BINARY_LENGTH = 262144; // 256KB = MAX_CODE_LENGTH + MAX_DATA_LENGT
 uint64_t MAX_CODE_LENGTH = 245760; // 240KB
 uint64_t MAX_DATA_LENGTH = 16384; // 16KB
 
-uint64_t ELF_HEADER_LEN = 120; // = 64 + 56 bytes (file + program header)
```

Comment:
This comment should not be lost. I suggest to keep it but rephrase it to e.g.:

// page-aligned ELF header with sufficient space to store 64 bytes for file header and 56 bytes for program header